src/ui/layout/mct-tree.vue

```diff
@@ -574,6 +595,11 @@ export default {
                 navigationPath
             };
         },
+        addMutable(mutableDomainObject, parentObjectPath) {
```

Comment:
Would there be ANY scenario that we'd need to check if the mutable exists first and destroy it or use the existing one?

Reply:
opted to err on the safe side and do `destroyMutableByPath` before adding one. The mutable shouldn't exist if we clean them up properly, but this will at least prevent memory leaks in case for any reason we haven't.